.github/workflows/ci.yml

```
@@ -230,7 +230,9 @@ jobs:
       - name: "Run Tests"
         run: "poetry run invoke unittest"
   changelog:
-    if: contains(fromJson('["develop","ltm-1.6"]'), github.base_ref)
+    if: |
+      contains(fromJson('["develop","ltm-1.6"]'), github.base_ref) ||
+      (github.head_ref == 'main' && github.base_ref == 'develop')
```

```suggestion
      contains(fromJson('["develop","ltm-1.6"]'), github.base_ref) &&
      (github.head_ref != 'main')
```

Shouldn't we ignore this check when merging from `main`?